projects/nano-editor.org/package.yml

```diff
@@ -0,0 +1,38 @@
+distributable:
+  url: https://www.nano-editor.org/dist/v{{ version.major }}/nano-{{ version.major }}.{{ version.minor }}.tar.gz
```

Comment:
You can use `nano-{{ version.marketing }}.tar.gz` or `{{ version.raw }}`, depending on which you think is correct. It's probably more correct to use `{{ version.raw }}` throughout.